tests/mtd_at25xxx/Makefile

```diff
@@ -0,0 +1,8 @@
+include ../Makefile.tests_common
+
+USEMODULE += mtd
+USEMODULE += at25xxx
```

Comment:
You won't need those anymore with the change to `Makefile.dep`.

Reply:
done